subsys/bluetooth/shell/has.c

```diff
@@ -75,6 +75,35 @@ static int cmd_preset_unreg(const struct shell *sh, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_has_register(const struct shell *sh, size_t argc, char **argv)
+{
+	int err;
+	struct bt_has_register_param param = {
+		.type = BT_HAS_HEARING_AID_TYPE_MONAURAL
+	};
+
+	for (size_t argn = 1; argn < argc; argn++) {
+		const char *arg = argv[argn];
+
+		if (strcmp(arg, "binaural") == 0) {
+			param.type = BT_HAS_HEARING_AID_TYPE_BINAURAL;
+		} else if (strcmp(arg, "banded") == 0) {
+			param.type = BT_HAS_HEARING_AID_TYPE_BANDED;
+		} else {
+			shell_help(sh);
+			return SHELL_CMD_HELP_PRINTED;
```

Comment:
You'd still need to handle `monaural` argument, even if it's default

Reply:
true 🤦‍♂️